rest-api/src/main/java/gov/cms/qpp/conversion/api/model/Constants.java

```diff
@@ -18,9 +18,9 @@
 	public static final String DYNAMO_CPC_ATTRIBUTE = "Cpc";
 	public static final String DYNAMO_CPC_PROCESSED_CREATE_DATE_ATTRIBUTE = "CpcProcessed_CreateDate";
 	public static final String DYNAMO_CREATE_DATE_ATTRIBUTE = "CreateDate";
-	public static final String CPC_PLUS_BUCKET_NAME_VARIABLE = "CPC_PLUS_BUCKET_NAME";
-	public static final String CPC_PLUS_FILENAME_VARIABLE = "CPC_PLUS_VALIDATION_FILE";
 	public static final String CPC_PLUS_UNPROCESSED_FILE_SEARCH_DATE_VARIABLE = "CPC_PLUS_UNPROCESSED_FILTER_START_DATE";
+	public static final String FMS_TOKEN = "";
+	public static final String AR_API_BASE_URL = "https://ar.qpp.internal/api";
```

Comment:
Does this URL not need to change based off environment?

Reply:
Fixed